Add web server configuration (#643)
SUSE/rmt: MANUAL.md

  * `mirroring.mirror_src`:
    Whether to mirror source (arch = `src`) RPM packages or not.
  * `mirroring.dedup_method`:

>     Whether to deduplicate files by hardlink or by copy, possible values are:

Reviewer: ```suggestion
    Whether to deduplicate files by a hardlink or copy. Possible values are:
```

    Determines proxy authentication mechanism, possible values are:
    `none`, `basic`, `digest`, `gssnegotiate`, `ntlm`, `digest_ie`, `ntlm_wb`

**Web server settings**

> The `web_server` section lets you tune the performance of the RMT server, by
> combining threads and workers to configure concurrency.

Reviewer: ```suggestion
The `web_server` section lets you tune the performance of your RMT server.
```

Not a blocking comment:

I don't think that we need to go too much into detail here.

  * `web_server.min_threads` setting:
    Specifies the minimum threads count a RMT server worker should spawn.
    Acceptable values: Integer greater than or equal to 1.
  * `web_server.max_threads` setting:
    Specifies the maximum threads count a RMT server worker should spawn.
    Acceptable values: Integer greater than or equal to 1.
  * `web_server.workers` setting:

>     Specifies the exact number of workers RTM web server should boot.

Reviewer: ```suggestion
    Specifies the number of web workers for RMT.
```

Reviewer: Typo in RMT (RTM) and we could simplify the sentence.
